package.json

```diff
@@ -41,6 +41,7 @@
     "gulp-sourcemaps": "^1.6.0",
     "gulp-uglify": "^1.4.2",
     "handlebars": "^4.0.3",
+    "localforage": "^1.4.0",
```

Comment:
We are only using localForage to store one key, `deviceId`; this seems a lot of overhead. Can we find something simpler here?